@LinProp/displayContributions.m

```diff
@@ -0,0 +1,125 @@
+function displayContributions(obj, part)
+
+    varName = inputname(1);
+
+    if numel(obj) ~= 1
+        error('Can only show budget for scalars');
+    end
+    
+    if obj.IsComplex
+        if nargin < 2
+            error('Input argument ''part'' not specified.');
+        end
+        
+        switch (part)
+            case 'real'
+                obj = real(obj);
+                varCall = 'real(%s)';
+            case 'imag'
+                obj = imag(obj);
+                varCall = 'imag(%s)';
+            otherwise
+                error('Input argument ''part'' must be ''real'' or ''imag''.');
+        end
+    else
+        varCall = '%s';
+    end
+
+    switch (obj.NetObject.Dependencies.Length)
+        case 0
+            fprintf('\n  Variable %s has no uncertainty contributions.\n', sprintf(varCall, varName));
+            return;
+        case 1
+            fprintf('\n  Variable %s has 1 uncertainty contribution.\n', sprintf(varCall, varName));
+        otherwise
+            fprintf('\n  Variable %s has %i uncertainty contributions.\n', sprintf(varCall, varName), obj.NetObject.Dependencies.Length);
+    end
+    
+    if exist('LoadVNATools', 'file')
+        LoadVNATools();
```

Comment:
This could yield into problems when the version of METAS UncLib and METAS VNA Tools are not the same. E.g.: `Metas.UncLib.Core.dll` of METAS VNA Tools could be older or newer than the already loaded assembly from METAS UncLib.

On our side this handled like that: 
- The first UncLib object is created without calling `LoadVNATools.m` first. In that case the UncLib assemblies are loaded from the METAS UncLib location.
- `LoadVNATools.m` is called before creating the first UncLib oject. In that case the UncLib assemblies are loaded from the METAS VNA Tools location and the `Metas.Vna.Data.UncIdDefs.GetInfluenceInfo2` exists.

See as well how I handle this in `unc_budget.m`.

Reply:
Oh, I never thought of that... Thank you for pointing this out.

I have pushed a commit that changes these lines to a try-catch, similar to the way you handled it in `unc_budget.m`.